stats/latencyhistogram15s32.go

```diff
@@ -29,8 +30,11 @@ func (l *LatencyHistogram15s32) Value(t time.Duration) {
 
 func (l *LatencyHistogram15s32) ReportGraphite(prefix, buf []byte, now time.Time) []byte {
 	snap := l.hist.Snapshot()
-	// TODO: once we can actually do cool stuff (e.g. visualize) histogram bucket data, report it
-	// for now, only report the summaries :(
+	for i, cnt := range snap[:len(snap)-1] {
+		bucket := l.hist.Limits[i]
+		buf = WriteUint32(buf, prefix, []byte(fmt.Sprintf("latency.histogram.%d", bucket)), cnt, now)
+	}
+	buf = WriteUint32(buf, prefix, []byte("latency.histogram.29000000"), snap[len(snap)-1], now)
```

Comment:
why is the last one hard coded to `29000000`? would be nice to have a comment explaining that